### PR TITLE
Update wget to fix travis builds

### PR DESCRIPTION
**Pull request by chetanmeh**
Fixes Invoker docker build failure by updating the `wget` version

## Description

Travis builds are currently failing (both [master][1] and PRs) due to failure in invoker docker build with following errors

```
Step 6/11 : RUN wget --no-verbose https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/bin/kubectl
 ---> Running in ddbe33270513
Connecting to storage.googleapis.com (173.194.198.128:443)
ssl_client: storage.googleapis.com: TLS connect failed
wget: error getting response: Connection reset by peer
The command '/bin/sh -c wget --no-verbose https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/bin/kubectl' returned a non-zero code: 1
```

However the build passes locally and there seems to be no error with storage.googleapis.com SSL as seen [here][2]. Looks like wget shipped with alpine:3.6 is [old has some issues][3] and updating that resolves the issue for us.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

[1]: https://travis-ci.org/apache/incubator-openwhisk/builds/385958097
[2]: https://travis-ci.org/chetanmeh/incubator-openwhisk/builds/386028169#L2356
[3]: https://github.com/gliderlabs/docker-alpine/issues/344#issuecomment-336065308

### CODE REVIEW

**Comment by csantanapr** [comment]
Hum no problem with not having a space in `wget&&` ?

File: common/scala/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ ENV PATH $JAVA_HOME/bin:$PATH
 
 
 RUN apk upgrade --update && \
-    apk add --update libstdc++ curl ca-certificates bash sed && \
+    apk add --update libstdc++ curl ca-certificates bash sed wget&& \
```
Comment (csantanapr) [inline common/scala/Dockerfile]: Hum no problem with not having a space in wget&& ?

Comment (chetanmeh) [reply common/scala/Dockerfile]: Looks like in hurry I forgot to add a space. However it does works and [wget gets updated](https://travis-ci.org/apache/incubator-openwhisk/jobs/386053710#L1980)

```
(12/13) Installing sed (4.4-r0)
(13/13) Installing wget (1.19.5-r0)
Executing busybox-1.26.2-r11.trigger
Executing ca-certificates-20161130-r2.trigger
```
Would fix this in a new PR

Comment (chetanmeh) [reply common/scala/Dockerfile]: Opened PR #3723

**Pull request merged**